src/book.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] rentals should be for getter and setter since we will also be adding the data and reading it from the UI.